src/_json.js

```diff
@@ -16,6 +16,36 @@ function parseJSON(json) {
   return JSON.parse(json, json_parse)
 }
 
+function parseJSONStreaming(json) {
+  var results = []
+
+  try {
+    results.push(parseJSON(json))
+  } catch (error) {
+    var position
+
+    /** JSON parse error message is throw by the browser */
+    if (error.message.match(/JSON.parse/g)) {
+      var matchResult = error.message.match(/column ([0-9+]*)/)
+      position = matchResult[1] - 1
+      /** Unexpected token message is throw by NodeJS */
+    } else if (error.message.match(/Unexpected token/g)) {
+      var matchResult = error.message.match(/at position ([0-9+]*)$/)
+      position = matchResult[1]
```

Comment:
Use the same strategy here.

Note that your capture group syntax seems off here. `\d` matches digits. I'm not sure why your character array includes `+`, nor why would accept 0 or more digits.

The capture group in my suggestion above seems better.

Reply:
Likewise, i think indexOf will be more efficient.

Reply:
Ewan is correct about the regex.. change it to `/at position ([0-9]+)/`

Reply:
The same of before, `/at position ([0-9]+)/` didn't work.